check/checksum.go

```diff
@@ -0,0 +1,182 @@
+package check
+
+import (
+	"bytes"
+	"crypto/md5"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/schollz/progressbar/v3"
+)
+
+func isHiddenFile(f string) bool {
+	return string(f[0]) == "."
```

Comment:
Não precisa converter de `char`

```suggestion
	return f[0] == '.'
```